### PR TITLE
Feature/bind to external element

### PR DESCRIPTION
**Pull request by miladkdz**
Regarding Issue #1

### CODE REVIEW

File: src/vaadin-tooltip.html [hunk 1]
```diff
@@ -35,6 +57,72 @@ <h1>Hello World!</h1>
...
+            target.addEventListener('focus', this.show.bind(this));
+            target.addEventListener('mouseleave', this.hide.bind(this));
+            target.addEventListener('blur', this.hide.bind(this));
+            target.addEventListener('tap', this.hide.bind(this));
+          }
+
+          this.addEventListener('mouseenter', this.hide.bind(this));
+        }
+
+        _detachFromTarget() {
```
Comment (peancored) [inline src/vaadin-tooltip.html]: Where do you use this function?

File: src/vaadin-tooltip.html [hunk 2]
```diff
@@ -13,9 +13,31 @@
...
+        display: block;
+        position: absolute;
+        outline: none;
+        z-index: 1000;
+        -moz-user-select: none;
+        -ms-user-select: none;
+        -webkit-user-select: none;
+        user-select: none;
+        cursor: default;
+
```
Comment (peancored) [inline src/vaadin-tooltip.html]: Extra newline

File: src/vaadin-tooltip.html (hunk 1 above)
Comment (miladkdz) [reply src/vaadin-tooltip.html]: Wrote it to call it at some point in the future. Do you suggest I remove it from this branch?

Comment (peancored) [reply src/vaadin-tooltip.html]: No, if you plan to use it, then it is ok

**Pull request merged**